### PR TITLE
Status icon to characters

### PR DESCRIPTION
**Pull request by harsh599**
Hello @lelouchB  I request you to kindly look into the problem as why my status images are breaking in FrontEnd side I am sure there must be something issue related to their access and the path of the images.
Looking forward to contribute more

### CODE REVIEW

File: frontend/src/pages/Components/Body/_Card.js [hunk 1]
```diff
@@ -15,6 +15,9 @@ function Card(props){
             </div>
             <div className="card__body">
                 <h3>{props.name}</h3>
+                {props.status == 'Alive' ?  <img src = '.img/alive.jpg' /> : null}
+                {props.status == 'Dead' ? <img src = '.img/dead.jpg' /> : null}
```
[the remark this answers is not in the document]
Comment (lelouchB) [reply frontend/src/pages/Components/Body/_Card.js:19]: Hey 
There is no dead status
It is deceased and deceased (possessed)

**Comment by lelouchB** [comment]
Hey @harsh599 
Have you tried using `useBaseUrl()` 
See this 
https://github.com/harsh599/final-space-api/blob/96b042e790d3454ebc9dad646ea5fdb81b914185/frontend/src/pages/Components/Body/_Card.js#L12
Also, there is no need for updating the status on the frontend now.

The issue was for adding it to the schema so it is included in the response and user don't have to check the status manually.
Once it has been updated on the backend, this change on the frontend will be much easier to incorporate.

If you want, you can send the status images to `backend/images/status` folder, so I can add it to the database. Or atleast I will approve it.

Let me know if you have any other questions.

Hope this answer your question.

**Comment by harsh599** [comment]
> Hey @harsh599
> Have you tried using `useBaseUrl()`
> See this
> https://github.com/harsh599/final-space-api/blob/96b042e790d3454ebc9dad646ea5fdb81b914185/frontend/src/pages/Components/Body/_Card.js#L12
> Also, there is no need for updating the status on the frontend now.
> 
> The issue was for adding it to the schema so it is included in the response and user don't have to check the status manually.
> Once it has been updated on the backend, this change on the frontend will be much easier to incorporate.
> 
> If you want, you can send the status images to `backend/images/status` folder, so I can add it to the database. Or atleast I will approve it.
> 
> Let me know if you have any other questions.
> 
> Hope this answer your question.

The Issue has been fixed now I think  I have been accessing the image in a wrong way
image

I have also added the images in the `backend/images/status` folder you can make the Back End Related changes.Thankyou Brother

**Comment by lelouchB** [comment]
Hey @harsh599 
Awesome 👍 
I will approve this PR but will have to wait for @ErickLemos to finish since he was first to take up the issue and it is assigned to him.

Great Work

**Review by lelouchB — APPROVED** [review]

**Comment by harsh599** [comment]
Hello @lelouchB  I have made some improvisations in the code for a better flexibility for you.

**Comment by harsh599** [comment]
> Hey @harsh599
> Awesome 👍
> I will approve this PR but will have to wait for @ErickLemos to finish since he was first to take up the issue and it is assigned to him.
> 
> Great Work

Oh Thankyou

**Comment by lelouchB** [comment]
> @lelouchB no problem, you can accept the PR and assign the task to it!

Just to be sure 
You are saying to reassign the issue
Right?

**Pull request closed without merging**